jarviscli/packages/timeIn.py

```diff
@@ -27,11 +39,22 @@ def main(self, s):
 
 
 def getLocation(s):
-    send_url = ("http://maps.googleapis.com/maps/api/geocode/json?address={0}&sensor=false".format(s)
-                )
+    file_path = module_path + '/../data/keys.json'
+    with open(file_path) as json_file:
+        data = json.load(json_file)
+    key = data['timein']
```

Comment:
```suggestion
    if 'timein' not in data or data['timein'] == 'insertyourkeyhere':
        print(Fore.RED + "Generate api key here: https://developers.google.com/maps/documentation/geocoding/start?hl=en_US")
        print(Fore.RED + "and add it to jarviscli/data/keys.json" + Fore.RESET)
        return None
    key = data['timein']
```

Would check if key is "insertyourkeyhere".

Probably not the "cleanest" solution but imagine user runs "time in" once, forgets / is too lacy to creating API key but runs "check time in" again - he would be asked to open issue instead creating an API key.

Would also replace os.path.isfile check from above.